### PR TITLE
Create CONTRIBUTING and onboarding document

### PR DESCRIPTION
**Pull request by malvikasharan**
Following my discussion with Achintya and RAM team lead Aida, I am creating documents required for the onboarding of new contributors to the white paper.

- [x] Contributing guideline
- [x] Onboarding checklist
- [x] Issue templates
- [x] Remove outdated information or empty files
- [x] Achintya to link the long list of projects in the onboarding document by the end of 5 July 2022

These documents will be helpful to get RAM team involved in contributor capacity.

### CODE REVIEW

**Comment by RaoOfPhysics** [comment]
Hi @malvikasharan. Can you please resolve the conflict as I can’t merge the PR or update it without problems?

File: .github/ISSUE_TEMPLATE/NEW_MEMBERS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# ASG White Paper Contributor onboarding - checklist
+
+- [ ] Connect with the ASG community manager (https://github.com/alan-turing-institute/asg-community#contact) who is coordinating the development of the ASG white paper and will be happy to guide you in your participation and contributions to the project.
+- [ ] Familiarise yourself with the project's goals (README) and contribution process (CONTRIBUTING)
+- [ ] Get access to the GitHub repository as a contributor
+- [ ] Get access to the Overleaf document where the white paper is currently being drafted
+- [ ] Make sure you have been given access to relevant Communications channels used for this project (the Turing Slack, coworking calls, ASG events)
+- [ ] Check the project's Zotero for reference: https://www.zotero.org/groups/4652021/turing_asg/library
+- [ ] Check out the [different kinds of roles and responsibilities](https://github.com/alan-turing-institute/asg-community/blob/main/documentation/asg-white-papers/contributor-roles.md) contributors of this project can take. We use the [CRediT Taxonomy](https://casrai.org/credit/) to define role beyond writing and reviewing the paper.
```
Comment (malvikasharan) [inline .github/ISSUE_TEMPLATE/NEW_MEMBERS.md:18]:
```suggestion
- [ ] Check out the [different kinds of roles and responsibilities](https://github.com/alan-turing-institute/asg-community/blob/main/documentation/contributor-roles.md) contributors of this project can take. We use the [CRediT Taxonomy](https://casrai.org/credit/) to define role beyond writing and reviewing the paper.
```

File: .github/ISSUE_TEMPLATE/NEW_MEMBERS.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Browse through currently open issues and Pull Requests to directly participate in the discussions as described in the [contributing guideline](https://github.com/alan-turing-institute/asg-community/blob/main/CONTRIBUTING.md)
+- [ ] Read the details onboarding documents to understand different routes to contributions, how your contributions will be acknowledged and resources that can help you get started as a contributor to the ASG white papers.
```
Comment (malvikasharan) [inline .github/ISSUE_TEMPLATE/NEW_MEMBERS.md:20]:
```suggestion
- [ ] Read the [onboarding document]((https://github.com/alan-turing-institute/asg-community/blob/main/onboarding.md)) to understand different routes to contributions, how your contributions will be acknowledged and resources that can help you get started as a contributor to the ASG white papers.
```

**Review by malvikasharan — COMMENTED** [review]
link update in issue template

File: onboarding.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+## Get in touch with the project team
+
+As a community-led project, there are several contributors to this project (see: https://github.com/alan-turing-institute/asg-community#who-is-involved).
+For onboarding and other questions related to this project, you can directly reach out to the Community Manager by emailing `arao@turing.ac.uk`, or contact via Slack.
+You can also create [a GitHub issue](https://github.com/alan-turing-institute/asg-community/issues/new) if you would like to discuss your idea openly on this repository.
+
+## Long list of projects
+
+The ASG Community Manager has compiled a long list of a project curated by ASG project leads and Research Application Managers. 
+Please browse through the information provided in this document (READ ONLY): [Link to be added by Achintya].
```
Comment (RaoOfPhysics) [inline onboarding.md:18]:
```suggestion
Please browse through the information provided [in this document](https://thealanturininstitute-my.sharepoint.com/:x:/g/personal/arao_turing_ac_uk/ETV-miIocs1Lv4Ss305GLG8B45egy4bx-NyP480dHBe3JQ?e=c6NbHU) (internal to the Turing).
```

**Review by RaoOfPhysics — APPROVED** [review]

**Review by RaoOfPhysics — APPROVED** [review]

**Pull request merged**